components/user-store/org.wso2.carbon.identity.user.store.configuration/pom.xml

```
@@ -122,7 +122,8 @@
                             org.wso2.carbon.utils; version="${carbon.kernel.package.import.version.range}",
                             org.wso2.carbon.identity.base; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.*; version="${carbon.identity.package.import.version.range}",
-                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}"
+                            org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
+                            org.apache.axiom.om
```

```suggestion
                            org.apache.axiom.om; version="${axiom.osgi.version.range}"
```
Add the version range for the package import.